harness/determined/pytorch/dsat/_run_dsat.py

```diff
@@ -54,14 +70,19 @@ def main(core_context: det.core.Context) -> None:
     args.config_path = os.path.basename(args.config_path)
     args.model_dir = os.path.basename(args.model_dir)
     args.include = [os.path.basename(p) for p in args.include] if args.include is not None else []
-    args.experiment_id = det.get_cluster_info()._trial_info.experiment_id
+    cluster_info = det.get_cluster_info()
+    if cluster_info is None or cluster_info._trial_info is None:
+        return
```

Comment:
If we just `return` in these cases, won't that lead to very confusing errors? Seems like an `assert` with an explanation that this needs to be run `on-cluster` would be more helpful.

Reply:
Sure that makes sense, I'll add it on a second pass